### PR TITLE
Added Michael to leaderboard

### PR DESCRIPTION
**Pull request by XenonMolecule**
Added the final validation loss from my best run: [from here](https://wandb.ai/michael-nlp/cs336_basics/reports/val_loss-24-04-17-14-39-11---Vmlldzo3NTg5MzU3)

Technically, the best the model achieved was 3.835, but I reported the final loss.

This particular model uses torch.compile, swiglu activations, and hyperparameter tuning (mostly learning rate, dropout, and weight decay.  For this particular run, I used the default architecture).

I also tried tied embeddings with/without swiglu, but never saw significant improvements.  For documentation of all runs, I have included the remaining results [here](https://api.wandb.ai/links/michael-nlp/am3gmzyj).

I'm running with a few more hyperparameter changes, and will update my position on the leaderboard accordingly if any of those work well, but for now 3.914 seems to be sticking.

### CODE REVIEW

**Comment by nelson-liu** [comment]
the link 404's for me, could you double-check the access permissions and update the link if necessary?

**Comment by XenonMolecule** [comment]
Sorry about that!  I tried exporting again and the link looks more familiar with the other ones in the repo: https://api.wandb.ai/links/michael-nlp/9hf4l35j

Let me know if this works, thanks Nelson!

**Comment by nelson-liu** [comment]
LGTM, thanks!
